src/agent/CMakeLists.txt

```diff
@@ -10,42 +10,33 @@ set(VCPKG_MANIFEST_DIR ${CMAKE_SOURCE_DIR}/../)
 
 project(Agent)
 
-include_directories(include)
-
 set(SOURCES
     src/agent.cpp
-    src/message_task.cpp
     src/task_manager.cpp
     src/register.cpp
+    src/signal_handler.cpp
 )
 
 set(HEADERS
```

Comment:
I think this is not necessary. If so, please also include `cmd_ln_parser.hpp`.

Reply:
done 05bf17df02fec16afd3a4954836dd16a48723f70